source/frontend/src/features/mapSideBar/acquisition/tabs/fileDetails/update/UpdateAcquisitionForm.tsx

```diff
@@ -266,6 +296,19 @@ const AcquisitionDetailSubForm: React.FC<{
             View={ContactInputView}
           ></ContactInputContainer>
         </SectionField>
+        {!isPersonSelected && (
```

Comment:
why not just:
ownerSolicitorContact?.organizationId && !ownerSolicitorContact?.personId 
or 
ownerSolicitorContact?.organizationId && !ownerSolicitorContact?.organizationId
instead of a state variable and a useffect?